test/engine/engine_support_test.cc

```diff
@@ -37,6 +37,121 @@ using ::testing::ContainsRegex;
 using ::testing::MatchesRegex;
 using ::testing::Pointwise;
 using ::testing::ElementsAreArray;
+using AngMomMatTest = MujocoTest;
+
+static constexpr char AngMomTestingModel[] = R"(
+<mujoco>
+<option integrator = "RK4" timestep = "0.001" jacobian = "dense"/>
+<worldbody>
+  <body name="link1">
```

Comment:
Most people will want to use this with mechanisms that have a free joint at the root, let's do that? Also, let's have a slider? So free at the base, then some arbitrary articulation with hinge, ball and slider.

As a side note, the angmom compute for mechanisms that are anchored to the world is in some senses "wrong" as in this case you want the angmom w.r.t the anchor, not w.r.t to the com...

Reply:
Updated the model. Added an inclined plane so that the mechanism falls on it and rolls down.